ci_framework/roles/dlrn_promote/tasks/get_hash_from_commit.yaml

```diff
@@ -0,0 +1,55 @@
+---
+- name: Get commit.yaml file
+  ansible.builtin.get_url:
+    url: "{{ commit_url }}/commit.yaml"
+    dest: "{{ cifmw_dlrn_promote_workspace }}/commit.yaml"
+    force: true
+  register: result
+  until: result is success
+  retries: 6
+  delay: 50
+
+- name: Load data from commit.yaml file
+  ansible.builtin.include_vars:
+    file: "{{ cifmw_dlrn_promote_workspace }}/commit.yaml"
+    name: _hashes
+
+- name: Get individual commit/distro hashes
```

Comment:
Why so many tasks for something that can be done in two tasks? One for setting the facts and one to print the debug message/variables.

Reply:
Done